src/components/tap-click/ripple.ts

```diff
@@ -14,92 +14,95 @@ export class RippleActivator extends Activator {
   }
 
   downAction(ev: UIEvent, activatableEle: HTMLElement, startCoord: Coordinates) {
-    let self = this;
-    if (self.disableActivated(ev)) {
+    if (this.disableActivated(ev)) {
       return;
     }
 
     // queue to have this element activated
-    self._queue.push(activatableEle);
-
-    nativeRaf(function() {
-      for (var i = 0; i < self._queue.length; i++) {
-        var queuedEle = self._queue[i];
-        if (queuedEle && queuedEle.parentNode) {
-          self._active.push(queuedEle);
-
-          // DOM WRITE
-          queuedEle.classList.add(self._css);
-
-          var j = queuedEle.childElementCount;
-          while (j--) {
-            var rippleEle: any = queuedEle.children[j];
-            if (rippleEle.tagName === 'ION-BUTTON-EFFECT') {
-              // DOM WRITE
-              rippleEle.style.left = '-9999px';
-              rippleEle.style.opacity = '';
-              rippleEle.style[CSS.transform] = 'scale(0.001) translateZ(0px)';
-              rippleEle.style[CSS.transition] = '';
-
-              // DOM READ
-              var clientRect = activatableEle.getBoundingClientRect();
-              rippleEle.$top = clientRect.top;
-              rippleEle.$left = clientRect.left;
-              rippleEle.$width = clientRect.width;
-              rippleEle.$height = clientRect.height;
-              break;
-            }
+    this._queue.push(activatableEle);
+
+    for (var i = 0; i < this._queue.length; i++) {
```

Comment:
since it is a scoped variable, I was worried that the compiler emits some boilerplate code to simulate the scope (like wrapping the for loop inside a function).

@adamdbradley does it make sense?